doc/operation-and-maintenance/Reloading-configuration-on-a-running-system.md

```diff
@@ -22,3 +27,147 @@ The following options' changes will be ignored when using `mongooseimctl` tool:
 * s2s_*
 * all_metrics_are_global
 * odbc_*
+
+
+### Node-specific options
+
+Very rarely we want different configs for each node in cluster.
+Than `reload_cluster` would detect configuration inconsistency and would not
```

Comment:
In such cases `reload_cluster` would detect a configuration inconsistency and would not allow configuration updates.

??